### PR TITLE
fix empty snapshot

### PR DESCRIPTION
**Pull request by ashleywang1**

### CODE REVIEW

**Comment by ashleywang1** [comment]
/kick gateway with subsets and upstream groups regression test flake

File: changelog/v1.3.15/fix-empty-snapshot.yaml [hunk 1]
```diff
@@ -1,7 +1,11 @@
 changelog:
   - type: FIX
     description: >
-      Pull in fix from solo-kit where GetStatusKeys was returning a slice with an empty string.
+      With disableProxyCollection: false (the default), extauth had a race condition
```
Comment (kdorosh) [inline changelog/v1.3.15/fix-empty-snapshot.yaml]:
```suggestion
      With `disableProxyCollection: false` (the default), extauth had a race condition
```

**Pull request merged**